packages/react-reconciler/src/ReactFiberReconcilerTypes.js

```diff
@@ -0,0 +1,369 @@
+  _debugIsCurrentlyTiming?: boolean,
+|};
+type PartialState<State, Props> =
+  | $Subtype<State>
+  | ((prevState: State, props: Props) => $Subtype<State>); // Callbacks are not validated until invocation
+type Callback = mixed;
+type Update<State> = {
+  expirationTime: ExpirationTime,
+  partialState: PartialState<any, any>,
+  callback: Callback | null,
+  isReplace: boolean,
+  isForced: boolean,
+  next: Update<State> | null,
+};
+type UpdateQueue<State> = {
+  // A processed update is not removed from the queue if there are any
+  // unprocessed updates that came before it. In that case, we need to keep
+  // track of the base state, which represents the base state of the first
+  // unprocessed update, which is the same as the first update in the list.
+  baseState: State, // For the same reason, we keep track of the remaining expiration time.
+  expirationTime: ExpirationTime,
+  first: Update<State> | null,
+  last: Update<State> | null,
+  callbackList: Array<Update<State>> | null,
+  hasForceUpdate: boolean,
+  isInitialized: boolean, // Dev only
+  isProcessing?: boolean,
+};
+type Batch = {
+  _defer: boolean,
+  _expirationTime: ExpirationTime,
+  _onComplete: () => mixed,
+  _next: Batch | null,
+};
+type FiberRoot = {
+  containerInfo: any, // Used only by persistent updates.
+  pendingChildren: any, // The currently active root fiber. This is the mutable root of the tree.
+  current: Fiber,
+  remainingExpirationTime: ExpirationTime, // Determines if this root can be committed.
+  isReadyForCommit: boolean, // A finished work-in-progress HostRoot that's ready to be committed.
+  // TODO: The reason this is separate from isReadyForCommit is because the
+  // FiberRoot concept will likely be lifted out of the reconciler and into
+  // the renderer.
+  finishedWork: Fiber | null, // Top context object, used by renderSubtreeIntoContainer
+  context: Object | null,
+  pendingContext: Object | null, // Determines if we should attempt to hydrate on the initial mount
+  +hydrate: boolean, // List of top-level batches. This list indicates whether a commit should be
+  // deferred. Also contains completion callbacks.
+  // TODO: Lift this into the renderer
+  firstBatch: Batch | null, // Linked-list of roots
+  nextScheduledRoot: FiberRoot | null,
+};
+export type Deadline = {timeRemaining: () => number};
+type OpaqueHandle = Fiber;
+type OpaqueRoot = FiberRoot;
+export type HostConfig<T, P, I, TI, HI, PI, C, CC, CX, PL> = {
+  getRootHostContext: (rootContainerInstance: C) => CX,
+  getChildHostContext: (parentHostContext: CX, type: T, instance: C) => CX,
+  getPublicInstance: (instance: I | TI) => PI,
+  createInstance: (
+    type: T,
+    props: P,
+    rootContainerInstance: C,
+    hostContext: CX,
+    internalInstanceHandle: OpaqueHandle,
+  ) => I,
+  appendInitialChild: (parentInstance: I, child: I | TI) => void,
+  finalizeInitialChildren: (
+    parentInstance: I,
+    type: T,
+    props: P,
+    rootContainerInstance: C,
```

Comment:
this needs to be updated from the source again, the method sig changed here with my recent PR

Reply:
Can you please point me to the change in your PR?

Reply:
the PR is merged! check master for the updated signature